shipengine_sdk/util/sdk_assertions.py

```diff
@@ -8,7 +9,83 @@
     RateLimitExceededError,
     ValidationError,
 )
-from shipengine_sdk.models import ErrorCode, ErrorSource, ErrorType
+from shipengine_sdk.models import Country, ErrorCode, ErrorSource, ErrorType
+
+validation_message = "Invalid address. Either the postal code or the city/locality and state/province must be specified."  # noqa
+
+
+def is_street_valid(street: List) -> None:
```

Comment:
Does the version of Python support Generic types?

Reply:
Yes, it does, I have gone through and added more in-depth type annotations since python 3.7 supports it. Things like `List[str]` here or `Dict[str, any]`, `Optional[ShipEngineConfig, Dict[str, any]]`, etc.

Reply:
This is in `commit: 5f14a32`.